scripts/populate-git-env.sh

```diff
@@ -0,0 +1,7 @@
+# List of environment variables you want to have access during next development and deployment.
```

Comment:
List of environment variables that are automatically populated, if not already set. 
Those variables are populated this way so that local deploys and local development behave similarly to staging and production.